src/sidebar/components/annotation-publish-control.js

```diff
@@ -15,6 +15,7 @@ import SvgIcon from '../../shared/components/svg-icon';
 /**
  * @typedef {import('../../types/api').Annotation} Annotation
  * @typedef {import('../../types/config').MergedConfig} MergedConfig
+ * @typedef {import('../util/theme').ThemeProperty} ThemeProperty
```

Comment:
Help me with a confusion! I see this `typedef` imported here, but how is it used?

Reply:
...I'm wondering if this doesn't need to be imported here...?

Reply:
Oh, indeed! This is left-over from an earlier version of this PR. I'll remove it.